btree/btree.c

```diff
@@ -1517,6 +1518,11 @@ static void node_move(struct nd *src, struct nd *tgt,
 	tgt->n_type->nt_move(src, tgt, dir, nr, tx);
 }
 
+static void node_capture(struct slot *slot, struct m0_be_tx *tx)
```

Comment:
Structure name and function name are same (`node_capture`). Please change one of these.

Reply:
Changed structure name to node_to_capture